package_policies/M17.md

```diff
@@ -1,8 +1,10 @@
 **R8.** Each xSDK member package should have sufficient documentation to support use and further development.
```

Comment:
This text needs updating as well, R8 => M17, right?

Reply:
Fixed.